### PR TITLE
codegen: Explicitly use glib::Value instead of importing

### PR DESCRIPTION
**Pull request by BrainBlasted**
Fixes an issue where glib::Value could conflict with other
things named "Value"

### CODE REVIEW

File: src/codegen/enums.rs [hunk 1]
```diff
@@ -354,7 +353,7 @@ impl FromGlib<{sys_crate_name}::{ffi_name}> for {name} {{
         writeln!(
             w,
             "impl<'a> FromValueOptional<'a> for {name} {{
-    unsafe fn from_value_optional(value: &Value) -> Option<Self> {{
+    unsafe fn from_value_optional(value: &glib::Value) -> Option<Self> {{
```
Comment (sdroege) [inline src/codegen/enums.rs:356]: This needs to use `use_glib_type("Value")` and that would also fix the CI (partially at least).

Same in the other places below

File: src/analysis/properties.rs [hunk 2]
```diff
@@ -209,7 +208,6 @@ fn analyze_property(
         }
         let set_bound = PropertyBound::get(env, prop.typ);
         if type_string.is_ok() {
-            imports.add("glib::Value");
             if set_bound.is_some() {
```
Comment (sdroege) [inline src/analysis/properties.rs:211]: clippy wants these two `if`s merged now

**Comment by sdroege** [comment]
Looks good to me otherwise, thanks!

File: src/analysis/properties.rs [hunk 3]
```diff
@@ -208,17 +207,14 @@ fn analyze_property(
...
-                        "Non nulable setter for property generated as nullable \"{}.{}\"",
-                        type_name, name
-                    );
-                }
+        if type_string.is_ok() && set_bound.is_some() {
+            imports.add("glib::object::IsA");
+            if !*nullable {
+                //TODO: support nonnulable setter if found any
+                warn!(
+                    "Non nulable setter for property generated as nullable \"{}.{}\"",
```
Comment (sdroege) [inline src/analysis/properties.rs:215]:
```suggestion
                    "Non nullable setter for property generated as nullable \"{}.{}\"",
```

File: src/analysis/properties.rs [hunk 4]
```diff
@@ -208,17 +207,14 @@ fn analyze_property(
...
-                    //TODO: support nonnulable setter if found any
-                    warn!(
-                        "Non nulable setter for property generated as nullable \"{}.{}\"",
-                        type_name, name
-                    );
-                }
+        if type_string.is_ok() && set_bound.is_some() {
+            imports.add("glib::object::IsA");
+            if !*nullable {
+                //TODO: support nonnulable setter if found any
```
Comment (sdroege) [inline src/analysis/properties.rs:213]:
```suggestion
                //TODO: support non-nullable setter if found any
```

**Comment by sdroege** [comment]
Not your typos, but as you've moved that code around let's just fix them now :) Otherwise good to me. @GuillaumeGomez?

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by sdroege** [comment]
Thanks!

**Pull request merged**